tests/functional/intelligent-host-selection/03-polling/flow.cylc

```diff
@@ -15,34 +17,36 @@ Tasks
     cycling mode = integer
     initial cycle point = 1
     [[graph]]
-        # Run good and mixed as controls
         R1 = """
             goodhosttask:start => stop_g
             mixedhosttask:start => stop_m
+            stop_m & stop_g => full_stop
```

Comment:
_DO NOT_ stop the workflow until everything is done.

Reply:
Wouldn't the workflow stop anyway?

Reply:
No - there's nothing handling the failed tasks - See update